lms/djangoapps/instructor/tests/test_proctoring.py

```diff
@@ -89,7 +94,9 @@ def test_proctoring_tab_visible_for_course_staff_and_admin(self, enable_proctore
         # verify that proctoring tab is visible for course instructor
         CourseStaffRole(self.course.id).remove_users(self.instructor)
         CourseInstructorRole(self.course.id).add_users(self.instructor)
-        self._assert_proctoring_tab_is_available()
+        response = self._assert_proctoring_tab_is_available()
```

Comment:
Nit: I know this isn't strictly related to your changes, but I figured I'd mention it. By reading the code, I'm gathering that the "proctored tab" is the "allowance section" tab. Is that right? When I read "proctored tab", I thought it was the "Special Exams" link that opens up the tabs, since there are multiple tabs on this page now, including the dashboard tab. I wonder if the test could be renamed to make it clearer.